hipster_dynamics/hipster_dynamics.ipynb

```diff
@@ -1,5 +1,15 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hipster Dynamics\n",
+    "Written by Philipp Rudiger<br>\n",
+    "Created: June 28, 2017<br>\n",
```

Comment:
Seems like sometimes it says "Created " and sometimes "Created: "; probably should always be the latter.